### PR TITLE
Added Snake Game

### PR DESCRIPTION
**Pull request by Atif0604**
Hey @chicken-biryani @harshita2216 
This PR will Resolve =>Add Snake Game #87
Please Review my PR

### CODE REVIEW

File: Snake-Game/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <br />
+    <br />
+
+    <div>
+        <canvas id="gameArea"></canvas>
+    </div>
+
+    <script src = './script.js'></script>
+</body>
+</html>
```
Comment (chicken-biryani) [inline Snake-Game/index.html:24]: Add a new line at the end of file.

Comment (chicken-biryani) [reply Snake-Game/index.html:24]: Use an html beautifier. Here use this link https://codebeautify.org/htmlviewer

File: Snake-Game/style.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    text-align: center;
+    padding: 5px 7px;
+}
+
+canvas{
+    background-color: white;
+    border-radius: 20px;
+    box-shadow: 3px 5px 6px black;
+    margin-right: 40px;
+}
```
Comment (chicken-biryani) [inline Snake-Game/style.css:28]: Add new line at end of file.

Comment (chicken-biryani) [reply Snake-Game/style.css:28]: Format code properly. Use this link:
https://codebeautify.org/jsviewer

**Comment by Atif0604** [comment]
Done @chicken-biryani

**Review by Ayushparikh-code — APPROVED** [review]
✅

**Comment by Atif0604** [comment]
Hey @Ayushparikh-code @chicken-biryani @harshita2216 
Please Add LGMSOC21 label to this PR that it should be showing in the label section of PR as my PR is not being counted just because the LGMSOC21 label is not showed up in label section

**Pull request merged**